runtimes/datascience/ubi9-python-3.9/Pipfile

```diff
@@ -27,7 +27,7 @@ mysql-connector-python = "~=8.3.0"
 ipykernel = "==6.13.0"
 ipython = "==8.10.0"
 ipython-genutils = "==0.2.0"
-jinja2 = "==3.0.3"
```

Comment:
Trivy does not complain about this on regular scheduled scans. It does see the package, https://github.com/opendatahub-io/notebooks/actions/runs/9867254845/job/27247380483#step:14:77, but it is not mentioned in https://github.com/opendatahub-io/notebooks/actions/runs/9867254845#summary-27247380483. I'm willing to guess that's because the severity of the cve is too low to show on the final report.

Reply:
Yes, this one is a `MEDIUM`. If it is more helpful, we can allow all severities in the daily report.

Reply:
If the goal is to allow comparing PR report and daily report to check cve went away, then all severities in all reports would need to be displayed.

Or we can say that we mostly fix the HIGH and CRITICAL stuff, so it's enough to show that.

Reply:
Initially, I thought that we were focusing only on HIGH and CRITICAL issues but, given these PRs, I would enable all types of severity on the daily report too.

Reply:
https://github.com/opendatahub-io/notebooks/pull/614